File: lab/c/pares/par_impar_ternario2.c
#include <stdio.h>

int ispar(int number){
    return number % 2 == 0;
}

const char* return_par_ou_impar(int number){
    return (ispar(number)) ? "Par" : "Ímpar";
}

int main(){
    printf("********************\n");
    for (int i=0; i <= 10; i++){
        printf("%d is %s\n", i, return_par_ou_impar(i));
        printf("********************\n");
    }
    return 0;
}

File: lab/javascript/pares/par_impar_ternario2.js
function ispar(number){
    return number % 2 === 0;
}


function return_par_ou_impar(number){
    return (ispar(number)) ? "Par" : "Ímpar";
}


console.log("********************");
for (let i=0; i <= 10; i++){
    console.log(`${i} is ${return_par_ou_impar(i)}`)
    console.log("********************");
}
